### PR TITLE
More minor spiffs

### PR DESCRIPTION
**Pull request by ferricoxide**
Gets RHEL scans above 97% when applied.

### CODE REVIEW

File: ash-linux/el7/STIGbyID/cat2/RHEL-07-030200.sls [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+#
+# CCI-001851 
+#    NIST SP 800-53 Revision 4 :: AU-4 (1) 
+#
+#################################################################
+{%- set stig_id = 'RHEL-07-030200' %}
+{%- set helperLoc = 'ash-linux/el7/STIGbyID/cat2/files' %}
+{%- set remoteCfg = '/etc/audisp/plugins.d/au-remote.conf' %}
+{%- set audSrv = salt.pillar.get('ash-linux:lookup:audisp-use-remote', 'yes') %}
+{%- set outpt = '/usr/bin/printf'%}
```
Comment (eemperor) [inline ash-linux/el7/STIGbyID/cat2/RHEL-07-030200.sls]: Can this line be removed?  `outpt` doesn't appear to be used anywhere.

File: ash-linux/el7/STIGbyID/cat2/RHEL-07-020100.sls [hunk 2]
```diff
@@ -34,11 +35,19 @@ script_{{ stig_id }}-describe:
 {%- for modFile in modFiles %}
   {%- if salt.file.search(modFile, '^[a-z]*\susb') %}
     {%- do foundMods.append(modFile) %}
-file_{{ stig_id }}-foundin-{{ modFile }}:
+file_{{ stig_id }}-foundin-{{ modFile }}-install:
   file.replace:
     - name: '{{ modFile }}'
-    - pattern: '^[a-z]*\susb.*$'
+    - pattern: '^\\[\\\\s\\]*install\\[\\\\s\\]*usb-storage\\[\\\\s\\]*/bin/true.*$'
```
Comment (eemperor) [inline ash-linux/el7/STIGbyID/cat2/RHEL-07-020100.sls]: Remove escaping and single quotes as with `active = ...` if possible.

File: ash-linux/el7/STIGbyID/cat2/RHEL-07-020100.sls [hunk 3]
```diff
@@ -34,11 +35,19 @@ script_{{ stig_id }}-describe:
...
     - repl: 'install usb-storage /bin/true'
+    - append_if_not_found: True
+    - backup: False
+file_{{ stig_id }}-foundin-{{ modFile }}-blacklist:
+  file.replace:
+    - name: '{{ modFile }}'
+    - pattern: '^\\[\\\\s\\]*blacklist\\[\\\\s\\]*usb-storage.*$'
```
Comment (eemperor) [inline ash-linux/el7/STIGbyID/cat2/RHEL-07-020100.sls]: Same.  Cleanup escaping and single quotes if possible.

File: ash-linux/el7/STIGbyID/cat2/RHEL-07-020100.sls [hunk 4]
```diff
@@ -38,14 +38,14 @@ script_{{ stig_id }}-describe:
...
-    - pattern: '^\\[\\\\s\\]*install\\[\\\\s\\]*usb-storage\\[\\\\s\\]*/bin/true.*$'
+    - pattern: ^[s]*install[s]*usb-storage[s]*/bin/true.*$
     - repl: 'install usb-storage /bin/true'
     - append_if_not_found: True
     - backup: False
 file_{{ stig_id }}-foundin-{{ modFile }}-blacklist:
   file.replace:
     - name: '{{ modFile }}'
-    - pattern: '^\\[\\\\s\\]*blacklist\\[\\\\s\\]*usb-storage.*$'
+    - pattern: '^[s]*blacklist[s]*usb-storage.*$'
```
Comment (eemperor) [inline ash-linux/el7/STIGbyID/cat2/RHEL-07-020100.sls]: Last nitpick...  Remove single quotes

Comment (ferricoxide) [reply ash-linux/el7/STIGbyID/cat2/RHEL-07-020100.sls]: Lulz... Only `s/\\//g` the file. p

**Pull request merged**